### PR TITLE
HackUPC 2018 edition

### PR DESCRIPTION
**Pull request by oriolclosa**
## What this PR does / Why we need it
1. Changed style according to the new edition.
2. Changed all information on the header.
3. FAQ changed, volunteer link removed, buildings added and travel reimbursement answer changed.
4. Added SVG loader with a PNG fallback for this edition.
5. Changed style with animations for the SVG loaded.
6. Changed sponsors and providers (moreover, the logos now are in separate folders).
7. Favicon and title changed.
8. 4 new URLs now available one with a legal document.
9. Cookie bar added.
10. Footer changed with links to the legal documents and new social networks. 

## Special notes for your reviewer (optional)
Gotta go fast :smile:!

## Some questions
- [X ] I have read the contributing guidelines
- [X ] I abide by this repository Code of Conduct

## Additional Notes (optional)
Do you want to add anything else? We :heart: to hear your opinions!
Not at the moment.

### CODE REVIEW

File: src/code_conduct/index.html [hunk 1]
```diff
@@ -37,25 +37,41 @@
...
+		    <li>Oscar Faixat, @faix in Slack, <a href="mailto:oscar@hackupc.com" target="_blank">oscar@hackupc.com</a>.</li>
+		</ul>
+		<p>Hackers@UPC reserves the right to revise, make exceptions to, or otherwise amend these policies in whole or in part.</p>
+
+		<hr>
+		<p>Last Reviewed: July 11, 2018.</p>
 	</div>
 
-	<div class="section-grey">
+	<div class="section-grey" style="box-shadow: 0 0 100px 0 #202326 !important;">
```
Comment (AsLogd) [inline src/code_conduct/index.html]: just why

File: src/legal_notice/index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+		<h3>8.MODIFICATION AND UPDATE</h3>
+		<p>The information contained in these pages is current as of the date of the last update.</p>
+		<p>HACKERS AT UPC reserves the right to modify this Legal Notice at any time and to restrict access temporarily or permanently, indicating in the heading of this Legal Notice the date of the last modification.</p>
+		<p>If you do not agree with any updates, please contact HACKERS AT UPC.</p>
+		
+		<hr>
+		<p>Last Reviewed: July 10, 2018.</p>
+	</div>
+
+	<div class="section-grey" style="box-shadow: 0 0 100px 0 #202326 !important;">
```
Comment (AsLogd) [inline src/legal_notice/index.html:101]: not cool

File: src/privacy_and_cookies/index.html [hunk 3]
```diff
@@ -234,6 +234,7 @@ <h4><strong>Which cookies do we use?</strong></h4>
 		    <td>Used to distinguish users.</td>
 		  </tr>
 		</table>
+		<p>We also use local storage to save if users have clicked on "OK" on the cookie notification bar. The data saved only consists of a key "cookies" with value "1".</p>
```
Comment (AsLogd) [inline src/privacy_and_cookies/index.html]: ask legal if we can add furhter descriptions or links about local storage

File: src/code_conduct/index.html [hunk 4]
```diff
@@ -37,25 +37,41 @@
...
+		<p>Event organizers will be happy to help participants contact campus security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the event. We value your attendance.</p>
+		<p>We expect participants to follow these rules at all hackathon venues, online interactions in relation to the event, hackathon-related social events, and on hackathon supplied transportation.</p>
+
+		<hr>
+		<h3>Reporting Procedures</h3>
+		<p>If you feel uncomfortable or think there may be a potential violation of the code of conduct, please report it immediately using one of the following methods. All reporters have the right to remain anonymous.</p>
+		<p>By sending us an email to contact@hackupc.com, your report will be received by our organizers. If you want to remain anonymous please send us an email through an anonymous email service or create a temporary email account.</p>
+		<p>If you need to contact a team member directly, please contact one of our organizers below.</p>
+		<ul style="padding-left: 1em;">
```
Comment (AsLogd) [inline src/code_conduct/index.html:63]: not cool

File: package.json [hunk 5]
```diff
@@ -23,6 +23,7 @@
     "nocache:index": "node node_modules/asset-cache-bust/bin/asset-cache-bust --asset-root dist dist/index.html",
     "nocache:live": "node node_modules/asset-cache-bust/bin/asset-cache-bust --asset-root dist dist/live.html",
     "scss:main": "node-sass --output-style compressed src/styles/main.2018f.scss dist/assets/css/main.2018f.min.css",
+    "scss:template": "node-sass --output-style compressed src/styles/template.scss dist/assets/css/template.min.css",
```
Comment (AsLogd) [inline package.json]: we probably shouldn't compile/copy template files

File: src/scripts/barcelona.ts [hunk 6]
```diff
@@ -83,7 +83,7 @@ class Barcelona{
 	private readonly _sky4StyleClass : string = ".st3";
 	private readonly _sunId : string = "lluna";
 	private readonly _skyId : string = "cel";
-	private readonly _buttonId : string = "boto";
+	private readonly _buttonId : string = "boto_2_";
```
Comment (AsLogd) [inline src/scripts/barcelona.ts]: a lot easier to replace id in the svg than change the .ts + .scss

File: package.json [hunk 7]
```diff
@@ -17,7 +17,9 @@
     "dist:scripts": "npm-run-all -p minjs:*",
     "dist:images": "imagemin-newer src/images dist/assets/img -p",
     "dist:html": "cp src/*.html dist",
+    "dist:favicon": "cp src/*.ico dist",
     "postdist:html": "npm-run-all -p nocache:*",
+    "nocache:favicon": "node node_modules/asset-cache-bust/bin/asset-cache-bust --asset-root dist dist/favicon.ico",
```
Comment (AsLogd) [inline package.json]: this won't do anything. asset-cache-bust finds urls inside html's to add a timestamp and thus avoiding cache

**Review by AsLogd — APPROVED** [review]
will work but we can fix some things

File: src/legal_notice/index.html (hunk 2 above)
Comment (casassg) [reply src/legal_notice/index.html:101]: Not at all.

File: src/code_conduct/index.html (hunk 4 above)
Comment (casassg) [reply src/code_conduct/index.html:63]: not as bad as the previous... but agree

**Pull request merged**